astropy/table/tests/test_showtable.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import os
+
+from ..scripts import showtable
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+ASCII_ROOT = os.path.join(ROOT, '..', '..', 'io', 'ascii', 'tests')
+FITS_ROOT = os.path.join(ROOT, '..', '..', 'io', 'fits', 'tests')
+VOTABLE_ROOT = os.path.join(ROOT, '..', '..', 'io', 'votable', 'tests')
+
+
+def test_missing_file(capsys):
+    showtable.main(['foobar.fits'])
+    out, err = capsys.readouterr()
+    assert err.startswith("ERROR: [Errno 2] No such file or directory: "
+                          "'foobar.fits'")
+
+
+def test_info(capsys):
+    showtable.main([os.path.join(FITS_ROOT, 'data/table.fits'), '--info'])
+    out, err = capsys.readouterr()
+    assert out == ('<Table length=3>{0}'
```

Comment:
For reference one can do `out.splitlines() == ['line 1', 'line2', etc..]` to get around the OS-dependence.  This is a bit easier for writing tests because you can write failing tests, run in the debugger, and then print `out.splitlines()` to get the right answer to paste into code (after verifying by hand that it seems right!).

Reply:
Yep, it's probably more handy for debugging, I will change.